Basemera/MyRecipes
# Add categories functionality

#### What does this PR do?
This PR adds functionality for categories

#### Description of Task to be completed?
This enables the CRUD functionality for categories.

#### How should this be manually tested?
Log in to the app.
You should be able to add, edit and delete categories. You will also be able to list all categories as well as view the details of a single category.
#### What are the relevant pivotal tracker stories?
[#159546011](https://www.pivotaltracker.com/story/show/159546011)
#### Any background context you want to add?
N/A
